static/app/utils/theme.tsx

```diff
@@ -199,67 +246,52 @@ const lightAliases = {
    */
   progressBackground: colors.gray100,
 
-  /**
-   * Background of alerts
-   */
-  alertBackgroundAlpha: 0.3,
-
-  /**
-   * Background of default badge (mainly used in NavTabs)
-   */
-  badgeBackground: colors.gray200,
```

Comment:
This is only used in `generateBadgeTheme` down below and is not used anywhere outside in the app. We can just explicitly write `colors.gray200` in `generateBadgeTheme` instead of aliasing it here.